lib/management/push.js

```diff
@@ -141,10 +146,7 @@ function startWSManagementClient(pushEndpoint, instanceId, token) {
               || 'localhost';
             const port = process.env.SECURE_CHANNEL_DEFAULT_FORWARD_TO_PORT
               || _config.port;
-            socket = net.createConnection({
-                host: host,
-                port: port,
-            });
+            socket = net.createConnection(host, port);
```

Comment:
The order is wrong here, its port then host (nodejs nicety). Why the change?

Reply:
wow... linter complains about the object form {port: value, host: value}

Reply:
Maybe it wants `{ host, port }` since the vars have the same name as the keys